hazelcast-jet-distribution/src/main/resources/bin/cluster.sh

```diff
@@ -60,21 +60,21 @@ if [ -z "$PORT" ]; then
 fi
 
 if [ -z "$GROUPNAME" ]; then
-    echo "No groupname is defined, running script with default groupname : 'dev'."
-    GROUPNAME="dev"
+    echo "No groupname is defined, running script with default groupname : 'jet'."
+    GROUPNAME="jet"
 fi
 
 if [ -z "$PASSWORD" ]; then
-    echo "No password is defined, running script with default password : 'dev-pass'."
-    PASSWORD="dev-pass"
+    echo "No password is defined, running script with default password : 'jet-pass'."
+    PASSWORD="jet-pass"
 fi
 
 if [ -z "$ADDRESS" ]; then
     echo "No specific ip address is defined, running script with default ip : '127.0.0.1'."
     ADDRESS="127.0.0.1"
 fi
 
-command -v curl >/dev/null 2>&1 || { echo >&2 "Cluster state script requires curl but it's not installed. Aborting."; exit -1; }
+command -v curl >/dev/null 2>&1 || { echo >&2 "Cluster state script requires curl but it's not installed. Aborting."; exit 255; }
```

Comment:
why not `1`?

Reply:
`shellchecker` told me that -1 isn't valid. Why not 255 then? :)

Reply:
seems like a non-standard error. 1 is typically used a a general error exit code. `255` means "exit status out of range"